tests/FSharp.Compiler.ComponentTests/FSharp.Compiler.ComponentTests.fsproj

```diff
@@ -58,7 +204,4 @@
     <ProjectReference Include="$(FSharpTestsRoot)\FSharp.Test.Utilities\FSharp.Test.Utilities.fsproj" />
```

Comment:
These tests can directly use FSharp.Compiler.Service.fsproj, and the test project can be included in FSharp.Compiler.Service.sln if not there already